curations/nuget/nuget/-/Aspose.PDF.yaml

```diff
@@ -3,9 +3,15 @@ coordinates:
   provider: nuget
   type: nuget
 revisions:
+  17.12.0:
+    licensed:
+      declared: OTHER
   21.10.1:
     licensed:
       declared: OTHER
   22.11.0:
     licensed:
       declared: OTHER
+  23.9.0:
```

Comment:
@capfei the Windows guy didn't notice the two files with same name but different casing 🤣 
I've merged versions from .pdf to .PDF as I realized notice generation is using the uppercase version.

I wanted to clean up so there was only one file but refrained from doing so. There are ~42 such duplicates as I see it, so I assume it is expected.